terraform/modules/k8s-karpenter/main.tf

```diff
@@ -19,6 +19,12 @@ serviceAccount:
   annotations:
     eks.amazonaws.com/role-arn: ${module.this[0].iam_role_arn}
 
+postInstallHook:
```

Comment:
Looks like the latest helm chart version doesn't have these values. Let's try to use the latest one.

Reply:
Changed helm chart version to `1.0.3` and remoed `postInstallHook`